### ISSUE TITLE
why?

### ISSUE THREAD

**Issue by igufei** [rogchap/v8go#10]
$ go get -u all
go: finding rogchap.com/v8go latest
go: downloading rogchap.com/v8go v0.0.0-20190917100500-0b5c09a84e8d
go: extracting rogchap.com/v8go v0.0.0-20190917100500-0b5c09a84e8d
# rogchap.com/v8go
In file included from v8go.cc:3:
C:/Users/Administrator/go/pkg/mod/rogchap.com/v8go@v0.0.0-20190917100500-0b5c09a84e8d/deps/include/v8.h:
7063:5: error: typedef 'v8::UnhandledExceptionCallback' is initialized (use decltype instead)
     _EXCEPTION_POINTERS* exception_pointers);
     ^~~~~~~~~~~~~~~~~~~
C:/Users/Administrator/go/pkg/mod/rogchap.com/v8go@v0.0.0-20190917100500-0b5c09a84e8d/deps/include/v8.h:
7063:5: error: '_EXCEPTION_POINTERS' was not declared in this scope
C:/Users/Administrator/go/pkg/mod/rogchap.com/v8go@v0.0.0-20190917100500-0b5c09a84e8d/deps/include/v8.h:
7063:26: error: 'exception_pointers' was not declared in this scope
     _EXCEPTION_POINTERS* exception_pointers);
                          ^~~~~~~~~~~~~~~~~~
C:/Users/Administrator/go/pkg/mod/rogchap.com/v8go@v0.0.0-20190917100500-0b5c09a84e8d/deps/include/v8.h:
8741:7: error: 'UnhandledExceptionCallback' has not been declared
       UnhandledExceptionCallback unhandled_exception_callback);
       ^~~~~~~~~~~~~~~~~~~~~~~~~~

Administrator@admin-PC MINGW64 /e/Work/Golang/test/v8go
$ go version
go version go1.13 windows/amd64

**Comment by rogchap** [comment]
related to #7 
tldr; No build of V8 for windows... yet; being worked on.
